modules/tracktion_engine/control_surfaces/types/tracktion_AlphaTrack.cpp

```diff
@@ -696,7 +696,7 @@ void AlphaTrackControlSurface::handleAsyncUpdate()
     if (flip)
         updateFlip();
 
-    if (auto edit = getEdit())
+    if (auto e = getEdit())
```

Comment:
I'm a bit surprised that the `Edit*` member isn't in the `protected` section instead of `private`...